assets/notifications.js

```diff
@@ -202,6 +203,8 @@ var Notifications = (function(options) {
         ret = $(html);
         ret.find('.notification-seen').click(function() {
             self.markSeen($(this).parents('.notification').data('id'));
+            $(this).parents('.notification').hide();
+            $('.notifications-icon-count, .notifications-header-count').text(parseInt($('.notifications-icon-count').html())-1);
```

Comment:
this is sooo AdminLTE oriented, we cannot have that hardcoded in the library
I also don't get the -1
and doesn't the code around line 300 handle that already ? (below // Update all counters)